src/esp/gfx/replay/Player.h

```diff
@@ -85,15 +85,32 @@ class AbstractPlayerImplementation {
           instances) = 0;
 
   /**
-   * @brief Set node transform
+   * @brief Set node transform from translation and rotation components.
    *
    * The @p handle is expected to be returned from an earlier call to
    * @ref loadAndCreateRenderAssetInstance() on the same instance.
    */
-  virtual void setNodeTransform(NodeHandle node,
+  virtual void setNodeTransform(const NodeHandle node,
                                 const Magnum::Vector3& translation,
                                 const Magnum::Quaternion& rotation) = 0;
 
+  /**
+   * @brief Set node transform.
+   *
+   * The @p handle is expected to be returned from an earlier call to
+   * @ref loadAndCreateRenderAssetInstance() on the same instance.
+   */
+  virtual void setNodeTransform(const NodeHandle node,
+                                const Mn::Matrix4& transform) = 0;
+
+  /**
+   * @brief Get node transform.
+   *
+   * The @p handle is expected to be returned from an earlier call to
+   * @ref loadAndCreateRenderAssetInstance() on the same instance.
+   */
+  virtual const Mn::Matrix4 getNodeTransform(const NodeHandle node) const = 0;
```

Comment:
Maybe rename this `hackGetNodeTransform` to convey that this accessor is part of our hack?